app/src/main/AndroidManifest.xml

```diff
@@ -14,6 +14,9 @@
         android:theme="@style/AppTheme"
         android:name=".App">
 
+        <meta-data android:name="com.kin.ecosystem.sdk.EnvironmentName" android:value="beta"/>
```

Comment:
It will be safer is name will be preset as a string resource 
```
<meta-data
    android:name="com.facebook.sdk.ApplicationId"
    android:value="@string/kinecosystem_enviorment_beta" />
```
You can define kinecosystem_enviorment_beta in strings.xml or from gradle

Reply:
👍